apps/src/code-studio/showProjectAdmin.js

```diff
@@ -24,15 +24,21 @@ export default () => {
   }
 
   if ($('#feature_project').length && dashboard.project.isProjectLevel()) {
-    var deleteUrl = `/featured_projects/${dashboard.project.getCurrentId()}`;
+    var unfeatureUrl = `/featured_projects/${dashboard.project.getCurrentId()}/unfeature`;
     $('#unfeature_project').click(function () {
       $.ajax({
-        url: deleteUrl,
-        type:'DELETE',
+        url: unfeatureUrl,
+        type:'PUT',
         dataType:'json',
+        data: {
+          featured_project: {
+            project_id: dashboard.project.getCurrentId(),
```

Comment:
Do we need the project id both the url and in the data payload?

Reply:
+1, it would be better to only specify the project id in one of these ways or the other.